src/components/Modals/NewCourse/CourseSelector.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <div class="autocomplete">
+    <input
+      v-bind:class="['search-box', searchBoxClassName]"
+      ref="dropdownInput"
+      v-model="searchText"
+      :placeholder="placeholder"
+      @keyup.esc="onEscape"
```

Comment:
This might be out of scope for this PR, but the escape key doesn't seem to close the course modal. This bug also exists on master so I don't know if we ever addressed it.

Reply:
fixed